pkg/kubernetes/cluster.go

```diff
@@ -54,7 +55,12 @@ func NewCluster(kubernetes *image.Kubernetes, configPath string) (*Cluster, erro
 		return &Cluster{ServerConfig: serverConfig}, nil
 	}
 
-	setMultiNodeConfigDefaults(kubernetes, serverConfig)
+	ip, err := netip.ParseAddr(kubernetes.Network.APIVIP)
+	if err != nil {
+		return nil, fmt.Errorf("parsing kubernetes APIVIP address: %w", err)
```

Comment:
Same as above. It's an error message and the user-friendly output shows `apiVIP`, so whatever you decide just be consistent between this and the previous message I commented on.